actionpack/lib/action_dispatch/middleware/server_timing_subscriber.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "active_support/subscriber"
+
+module ActionDispatch
+  class ServerTimingSubscriber < ActiveSupport::Subscriber
```

Comment:
Should we allow this `Subscriber` and the `subscribe` cal in `ActiveSupport::Notifications.subscribe` to handle monotonic timing? This is doable but would require some refactoring on the `ActiveSupport::Notifications` to  handle monotonic timing.

I would just like to get some feelers on wether this is ❤️ or 💔